build/preview.js

```diff
@@ -32,7 +37,7 @@ const cacheOptions = {
   cacheFile: `${__dirname}/.browserify-cache.json`,
 };
 
-gulp.task('preview:clean', () => require('del')([ASSETS_FOLDER]));
+gulp.task('preview:clean', () => require('del')([PUBLIC_FOLDER]));
```

Comment:
this was used to have `/public/images` symlinks for map files to be loaded. please make sure that still works.